repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -125,3 +127,8 @@ object ProjectUpdateFileRequest {
       unlift(ProjectUpdateFileRequest.unapply)
     )
 }
+
+final case class DummyResponseBody(dummyJsonParameter: String)
```

Comment:
I'm going to ask you the same question as before.
Are you sure that field with a name `dummyJsonParameter` exists in every place where you use this `DummyResponseBody` ?

Reply:
This is just a dummy parameter in dummy object. We can delete gitlab methods implementations in GitLabProjectVersioning and thair tests and forget about this, cause this is not fair implementations but just simulations

Reply:
I don't understand, why we should delete methods in `GitLabProjectVersioning` or tests.

Reply:
dummy is just a Dummy, this parameter and object is only used now by GitLabProjectVersioning and GitLabProjectVersioningTest, due to the abstractness of these class methods

Reply:
You are right, but I'm struggling to understand how does it related to my original question.
```suggestion
final case class DummyResponseBody()
```
That's what I wanted from you. An empty class without fields. In that case, we won't fail if the response from real GitLab services does not contain `test` or `dummyJsonParameter` fields. As far as I remember, I told you to make such an empty class for it when we discussed it in Google Meetings.

Reply:
the only reason why this dummuobject are not in test utils package is that it used in simulation-methods post and put (GitLabProjectVersioning class)

Reply:
Done!